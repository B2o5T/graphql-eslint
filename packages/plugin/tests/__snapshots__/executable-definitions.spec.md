// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
❌ Error

    > 1 | type Query { t: String }
        | ^^^^ The "Query" definition is not executable.
`;
