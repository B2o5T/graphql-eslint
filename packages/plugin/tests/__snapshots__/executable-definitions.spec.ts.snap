// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
> 1 | type Query { t: String }
    | ^ The "Query" definition is not executable.
`;
